sourcegraph/sourcegraph
# [SG-43524]: Fix edit insight repository select input dropdown on keyboard naviagtion

### Description
- When using the keyboard to navigate through the repositories and select input, each of the selected items when active shows a different background color. [Ref here](https://k8s.sgdev.org/insights/edit/aW5zaWdodF92aWV3OiIyMjZENHJKUVd3RkpFWktMYmlWeWFHNmtjenQi?dashboardId=all)
- Panel Loading state inherits wrong background color for theme state.

### Refs
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/43524)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-43524)

### Expected Behavior
Should display similar background color as in the mouse hover state

### Test Plan
1. Navigate to the insight edit page.
2. Using the keyboard, trigger open the repositories select input
3. Navigate up and down, and verify that when the select item is active, the background color is the same as hover state bg-color
4. Verify that the loading state shows the right dropdown background color.

### Demo

https://user-images.githubusercontent.com/45232708/198271168-69f088f4-bf05-4458-bddd-c2c3d0e7eeba.mov


## App preview:

- [Web](https://sg-web-contractors-sg-43524.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
